projects/instant-ui-library/src/lib/shared/models/color.model.ts

```diff
@@ -0,0 +1,7 @@
+export type ModeType = 'primary' | 'secondary' | 'tertiary' | 'highlight';
```

Comment:
`export type ModeType = Mode.PRIMARY | Mode.SECONDARY | Mode.TERTIARY | Mode.HIGHLIGHT;`

Reply:
This was marked resolved?

Reply:
Oops